### PR TITLE
chore: ensure waitFor is given

### PR DESCRIPTION
**Pull request by makelicious**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: packages/events/src/service/events/actions/approve-correction.ts
Comment (makelicious) [inline packages/events/src/service/events/actions/approve-correction.ts:1]: unused

File: packages/events/src/service/events/actions/reject-correction.ts
Comment (makelicious) [inline packages/events/src/service/events/actions/reject-correction.ts:1]: unused

**Review by rikukissa — APPROVED** [review]

File: packages/events/src/router/event/index.ts [hunk 1]
```diff
@@ -210,6 +213,7 @@ export const eventRouter = router({
 
       const updatedEvent = await processAction(
         {
+          waitFor: input.waitFor ?? true,
```
Comment (Zangetsu101) [inline packages/events/src/router/event/index.ts]: Do we want to wait for elasticsearch to index the event by default during "READ" action? Raising this because in <1.9, this was probably one of the more expensive operations during a record download. 

Let's break it down a bit more, we are adding a "READ" action to the record during this request, does it change the index in such a way that client needs to be aware of immediately?

File: packages/client/src/v2-events/features/events/actions/dedup/DuplicateForm.tsx [hunk 2]
```diff
@@ -82,7 +82,8 @@ export const DuplicateForm = ({ eventIndex }: { eventIndex: EventIndex }) => {
           actions.duplicate.markNotDuplicate.mutate({
             transactionId: getUUID(),
             eventId: eventIndex.id,
-            keepAssignment: true
+            keepAssignment: true,
+            waitFor: true
```
Comment (Zangetsu101) [inline packages/client/src/v2-events/features/events/actions/dedup/DuplicateForm.tsx]: Does the `markNotDuplicate` call change the index in a way that client immediately needs to be aware of? I'm just concerned about the performance impact this might have, unless absolutely necessary (e.g. actions that unassign the user after completion), we should use this sparingly I believe
